client/sites/settings/administration/index.jsx

```diff
@@ -0,0 +1,41 @@
+import { CompactCard } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import { useSelector } from 'react-redux';
+import SettingsSectionHeader from 'calypso/my-sites/site-settings/settings-section-header';
+import isSiteWpcomStaging from 'calypso/state/selectors/is-site-wpcom-staging';
+import { getSelectedSiteId, getSelectedSiteSlug } from 'calypso/state/ui/selectors';
+import SiteTools from '../../../my-sites/site-settings/site-tools';
+import { SOURCE_SETTINGS_ADMINISTRATION } from '../../../my-sites/site-settings/site-tools/utils';
+
+export default function AdministrationSettings() {
+	const translate = useTranslate();
+	const isWpcomStagingSite = useSelector( ( state ) =>
+		isSiteWpcomStaging( state, getSelectedSiteId( state ) )
+	);
+	const slug = useSelector( getSelectedSiteSlug );
+
+	return (
+		<div className="administration-settings">
+			{ ! isWpcomStagingSite && (
+				<SiteTools
+					headerTitle={ translate( 'Administration' ) }
+					source={ SOURCE_SETTINGS_ADMINISTRATION }
+				/>
+			) }
+			{ isWpcomStagingSite && (
+				<div>
+					<SettingsSectionHeader title={ translate( 'Administration' ) } />
+					<CompactCard>
+						<p>
+							{ translate( 'Manage staging sites from the {{a}}staging dashboard{{/a}}.', {
+								components: {
+									a: <a href={ `/staging-site/${ slug }` } />,
+								},
+							} ) }
+						</p>
+					</CompactCard>
```

Comment:
Needs design work but should be ok for now - nobody should really be seeing this.

Reply:
I see... this is because the path is registered for all site types.

I wonder if the more appropriate approach is for `<SidebarItem/>` to redirect to `/overview` if the path is not supported. This is what we've been doing in site management panel, e.g. when non-atomic sites are trying to access some dev-tool pages.

This can be followed up later if needed 👍

Reply:
Let's follow up, would prefer to avoid redirects where possible, better to not link in the first place or show the site selector when the attempted site isn't supported.